site/public/js/forum.js

```diff
@@ -1105,3 +1106,34 @@ function loadThreadHandler(){
         });
     });
 }
+
+var filters_applied = [];
```

Comment:
Is this supposed to be within loadThreadHandler? Or in a function at all? If my understanding is correct, the intent is not to have this within a function? I'm mostly asking for clarification here since the merge conflict relates to this chunk.

Reply:
Yes, the idea was to have it global to make it accessible to multiple pages!